java-client/src/test/java/org/opensearch/client/opensearch/model/BehaviorsTest.java

```diff
@@ -60,22 +58,6 @@ public void testSingleKeyDictionary() {
         assertEquals("some-value", q.value().stringValue());
     }
 
-    @Test
-    public void testAdditionalPropertyOnClass() {
```

Comment:
@kmessaoudi why this test was removed?

Reply:
Because it's linked to ShapeQuery that has been removed

Reply:
I can adapt it to XyShapeQuery if you prefer ?

Reply:
OK I will adapt it then

Reply:
Done